app/models/provider_enrichment.rb

```diff
@@ -77,4 +82,12 @@ def set_provider_code
     # Note: provider_code is only here to support c# counterpart, until provide_code is removed from database
     self.provider_code = provider.provider_code if provider_code.blank?
   end
+
+  def email_present?
+    !email.nil?
```

Comment:
```suggestion
    email.present?
```

Reply:
Also: `allow_nil: true` on the validation.